src/recipe.rs

```diff
@@ -115,7 +127,8 @@ impl<'src, D> Recipe<'src, D> {
           io_error: error,
         })?;
       let mut path = tmp.path().to_path_buf();
-      path.push(self.name());
+      let suffix = Platform::get_script_file_suffix(interpreter);
```

Comment:
Since this isn't platform specific, we can just inline it:

```suggestion
      let suffix = if command.ends_with("powershell")
        || command.ends_with("powershell.exe")
      {
        ".ps1"
      } else {
        ""
      };
```